src/AfricasTalkingChannel.php

```diff
@@ -47,7 +47,7 @@ public function send($notifiable, Notification $notification)
         }
 
         try {
-            $this->africasTalking->sms()->send($params);
+            return $this->africasTalking->sms()->send($params);
```

Comment:
merged to https://github.com/laravel-notification-channels/africastalking/pull/17